doc/radosgw/multisite.rst

```diff
@@ -0,0 +1,710 @@
+=============
+RGW Multisite
+=============
+
+.. versionadded:: Jewel
+
+From Ceph release Jewel and beyond, you may configure each :term`Ceph Object
+Gateway` to work in an active active zone configuration, allowing for writes to
+non-master zones. Following are the basic terminologies that would be used:
+
+- **Zone**: A zone is *logical* grouping of one or more Ceph Object Gateway
+  instances. There will be one Zone that should be designated as the master zone
+  in a zonegroup, which will handle all bucket and user creation.
+
+- **Zonegroup**: A zonegroup consists of multiple zones, this approximately
+  corresponds to what used to be called as a region in pre Jewel releases for
+  federated deployments. There should be a master zonegroup that will handle
```

Comment:
Would this section just replace the section about federated deployments at some point in time? Otherwise that should at least get a warning that it doesn't work anymore with Jewel, it fails when there is no realm defined.